tests/Events/UpdateEventTest.php

```diff
@@ -23,6 +25,25 @@ public function testUpdateChartKey()
         self::assertNotNull($retrievedEvent->updatedOn);
     }
 
+    public function testUpdateIsInThePast()
+    {
+        $chart = $this->seatsioClient->charts->create();
+        $season = $this->seatsioClient->seasons->create($chart->key, (new SeasonCreationParams())->setEventKeys(['event1', 'event2']));
+
+        $this->seatsioClient->events->update('event1', UpdateEventParams::create()->setIsInThePast(true));
+        $retrievedEvent = $this->seatsioClient->events->retrieve('event1');
+        self::assertTrue($retrievedEvent->isInThePast);
+        self::assertNotNull($retrievedEvent->updatedOn);
+
+        try {
```

Comment:
I wouldn't test events in the past being immutable - that's covered by the tests in `seatsio-core-v2`.